src/google_apps_clj/credentials.clj

```diff
@@ -87,10 +89,41 @@
       (.setRefreshToken (:refresh-token auth-map))
       (.setTokenType (:token-type auth-map)))))
 
-(t/ann ^:no-check build-credential [GoogleCtx -> HttpRequestInitializer])
+(t/ann credential-with-scopes [GoogleCredential (t/Seq t/Str) -> GoogleCredential])
+(defn ^GoogleCredential credential-with-scopes
+  "Creates a copy of the given credential, with the specified scopes attached.
+  `scopes` should be a list or vec of one or more Strings"
+  [^GoogleCredential cred, scopes]
+  (.createScoped cred (set scopes)))
+
+(t/ann default-credential (t/Fn [-> GoogleCredential] [(t/Seq t/Str) -> GoogleCredential]))
+(defn ^GoogleCredential default-credential
+  "Gets the default credential as configured by the GOOGLE_APPLICATION_CREDENTIALS environment variable
+  (see https://developers.google.com/identity/protocols/application-default-credentials)
+  Optionally you may specify a collection (list/vec/set) of string scopes to attach to the credential"
+  ([]
+   (GoogleCredential/getApplicationDefault))
+  ([scopes]
```

Comment:
The only reason I'd be wary of this convenience arity is if there were an alternate way of specifying the account credentials other than the env var; this would be the natural place to specify it. I couldn't find anything in 30 seconds of searching though. Safe to dismiss this concern, you reckon?